package.json

```
@@ -94,7 +94,7 @@
     "**/flat": "^5.0.2",
     "**/follow-redirects": "^1.15.2",
     "**/front-matter": "^4.0.2",
-    "**/glob-parent": "^6.0.0",
+    "**/glob-parent": "^6.0.1",
```

Note: The CVE was patched in 6.0.1.

---

@ananzh Why not just make the resolution be?
```suggestion
    "**/glob-parent": "^6.0.2",
```